src-ui/components/multi/MappingPane.cpp

```diff
@@ -1452,6 +1452,9 @@ struct SampleDisplay : juce::Component, HasEditor
     std::unique_ptr<MyTabbedComponent> waveformsTabbedGroup;
     size_t selectedVariation{0};
 
+    std::string filePattern{"*.wav;*.mp3;*.aif"};
```

Comment:
In src/browser there’s a function of surfed file thpes. Can we at least comment there to update here if we add one (and vice versa) or better add a method in browser to make this string?

Reply:
of course